helpdesk/templates/helpdesk/ticket_dependency_del.html

```diff
@@ -7,7 +7,7 @@
     <a href="{% url 'helpdesk:list' %}">{% trans "Tickets" %}</a>
 </li>
 <li class="breadcrumb-item">
-    <a href="{% url 'helpdesk:list' %}{{ ticket.id }}/">{{ ticket.queue.slug }}-{{ ticket.id }}</a>
+    <a href="{% url 'helpdesk:list' %}{{ dependency.ticket.id }}/">{{ dependency.ticket.queue.slug }}-{{ dependency.ticket.id }}</a>
```

Comment:
Good catch, I didn't see that ticket was not available in the template. You can replace `{% url 'helpdesk:list' %}{{ dependency.ticket.id }}/` by `{{ dependency.ticket.get_absolute_url }}` 😉